Error in test_training.test_pathmind_env_module()

`AttributeError: 'PathmindEnvironment' object has no attribute 'action_space'`


More context
```
   @pytest.mark.integration
    def test_pathmind_env_module():
        ray.shutdown()
        output_dir = f"testoutputs/test-pathmind-env-module-{randint(0,1000)}"
        run.main(
            environment="tests.cartpole.PathmindEnvironment",
            max_episodes=1,
            is_pathmind_simulation=True,
>           output_dir=output_dir,
        )

nativerl/python/tests/test_training.py:46: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
nativerl/python/run.py:155: in main
    env_instance = env_creator(env_config=env_config)
nativerl/python/pathmind_training/environments.py:114: in __init__
    self.action_space = self.define_action_space()
nativerl/python/pathmind_training/environments.py:131: in define_action_space
    action_space = self.nativeEnv.getActionSpace(i)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <pathmind_training.environments.get_native_env_from_simulation.<locals>.PathmindEnv object at 0x155d23750>, agent_id = 0

    def getActionSpace(self, agent_id=0):
>       space = self.simulation.action_space(agent_id=agent_id)
E       AttributeError: 'PathmindEnvironment' object has no attribute 'action_space'

nativerl/python/pathmind_training/environments.py:383: AttributeError

```

--- a/nativerl/python/pathmind_training/pynativerl.py
+++ b/nativerl/python/pathmind_training/pynativerl.py
@@ -1,3 +1,4 @@
+import importlib
 from abc import ABC, abstractmethod
 from typing import List, Optional, Union
 
@@ -92,3 +93,20 @@ class Environment(ABC):
     @abstractmethod
     def getRewardTerms(self, agent_id: int = 0) -> np.array:
         return NotImplemented
+
+
+def get_environment_class(env_name):
+    """Get environment class instance from a string, interpreted as Python module
+    :param env_name:
+    :return:
+    """
+    class_name = env_name.split(".")[-1]
+    module = env_name.replace(f".{class_name}", "")
+    lib = importlib.import_module(module)
+    return getattr(lib, class_name)
+
+
+def createEnvironment(env_name):
+    clazz = get_environment_class(env_name)
+    obj = clazz()
+    return obj

--- a/nativerl/python/tests/test_training.py
+++ b/nativerl/python/tests/test_training.py
@@ -37,14 +37,12 @@ def test_freezing():
 
 
 @pytest.mark.integration
-@pytest.mark.xfail(reason="Still debugging this")
 def test_pathmind_env_module():
     ray.shutdown()
     output_dir = f"testoutputs/test-pathmind-env-module-{randint(0,1000)}"
     run.main(
         environment="tests.cartpole.PathmindEnvironment",
         max_episodes=1,
-        is_pathmind_simulation=True,
         output_dir=output_dir,
     )
 

